app/assets/javascripts/application.js

```diff
@@ -1,10 +1,10 @@
-// This is a manifest file that'll be compiled into application.js, which will include all the files
+// This is a manifest file that"ll be compiled into application.js, which will include all the files
 // listed below.
 //
-// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, or any plugin's
+// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, or any plugin"s
 // vendor/assets/javascripts directory can be referenced here using a relative path.
 //
-// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
+// It"s not advisable to add code directly here, but if you do, it"ll appear at the bottom of the
 // compiled file. JavaScript code in this file should be added after the last require_* statement.
 //
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
```

Comment:
Changes it this file are not needed

Reply:
I can see the global 'replace all' was not a good idea.. 😞 but after that I hope to be ok with Rubocop.
Thanks I will fix this and merge.